app/components/node-navbar/template.hbs

```diff
@@ -0,0 +1,49 @@
+{{#ember-wormhole to='secondaryNavbar' renderInPlace=renderInPlace}}
+    <nav class='navbar osf-project-navbar' role='navigation'>
+        <div class='container'>
+            <div class='navbar-header'>
+                <button type='button' class='navbar-toggle collapsed' data-toggle='collapse' data-target='.project-nav'>
+                    <span class='sr-only'>{{t 'node_navbar.toggle'}}</span>
+                    <span class='fa fa-bars fa-lg'></span>
+                </button>
+                <span class='navbar-brand visible-xs visible-sm'>
+                    {{t 'node_navbar.project_nav'}}
+                </span>
+            </div>
+            <div class='collapse navbar-collapse project-nav'>
+                <ul class='nav navbar-nav'>
+                    <li class="{{if (eq activeTab 'title') 'active'}}">
```

Comment:
If the linked page doesn't exist in the app, there's no point worrying about setting `class="active"`. If it does exist in the app, it'd be better to use the route name than a redundant enum of route keys.

It might help simplify things to add a little nested component for these links:
```hbs
{{! node-navbar/template.hbs }}
<ul class='nav navbar-nav'>
  {{#node-navbar/link node=node routeName='guid-node'}}
    {{node.title}}
  {{/node-navbar/link}}
  {{#node-navbar/link node=node hrefPath='/files/'}}
    {{t 'node_navbar.files'}}
  {{/node-navbar/link}}
  ...
</ul>
```
```ts
// node-navbar/link/component.ts
@tagName('li')
class NodeNavbarLink extends Component {
  @service router;

  @className
  @equal('routeName', 'router.currentRouteName') active;
}
```
```hbs
{{! node-navbar/link/template.hbs }}
{{#if routeName}}
  {{#link-to routeName node.id}}
    {{yield}}
  {{/link-to}}
{{else}}
  <a href="/{{node.id}}{{hrefPath}}">
    {{yield}}
  </a>
{{/if}}
```
Does that seem reasonable?

Reply:
It seems fine. And I really like the use of `router.currentRouteName`. The one thing I am not a big fan of is the use of routename/hrefpath as a way to determine link-to/a-tag. It seems to hide behind those variables whether we are supporting something, and also will have perpetual support for non link-tos, which will eventually not use.

Reply:
Trying to think of a way to reconcile being very explicit about normal links and link-tos, and use router.currentRouteName